index.html

```diff
@@ -183,32 +184,78 @@ <h2>
         </h2>
-        <form action="" class="form-fields flex-column">
-          <div class="input-item">
-            <label for="name">
-              <input
-                type="text"
-                id="name"
-                name="user_name"
-                placeholder="Full name"
-                maxlength="30"
-                required
-              />
-            </label>
-          </div>
-          <div class="input-item">
-            <label for="mail">
-              <input
-                type="email"
-                id="mail"
-                name="user_email"
-                placeholder="Email address"
-                required
-              />
-            </label>
-          </div>
+        <form
+          action="https://formspree.io/f/xgebnejv"
+          method="POST"
+          class="form-fields flex-column"
+        >
+          <label for="name">
+            <input
+              type="text"
+              id="name"
+              name="user-name"
+              maxlength="30"
+              placeholder="Full name"
+              required
+            />
+          </label>
+          <label for="mail">
+            <input
+              type="email"
+              id="mail"
+              name="user-email"
+              maxlength="30"
+              placeholder="Email address"
+              required
+            />
+          </label>
+          <label for="message">
+            <textarea
+              name="user-message"
+              id="message"
+              cols="30"
+              rows="10"
+              placeholder="Hey I’d like to connect to..."
+              maxlength="500"
+              required
+            ></textarea>
+          </label>
+          <button class="form-btn btn" type="submit">Get in touch</button>
         </form>
       </section>
+      <!-- Footer section -->
+      <footer class="footer-section flex-column">
+        <a href="https://mail.google.com/mail/u/0/#inbox?compose=new"
+          >viktorsb14@gmail.com</a
+        >
+        <ul class="social-media flex-row">
+          <li>
+            <a href="https://twitter.com/VctorMC20"
+              ><img src="icons/twitter.svg" alt="twitter icon"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/victormc13/"
+              ><img src="icons/Linkedin icon.svg" alt="linkedin icon"
+            /></a>
+          </li>
+          <li>
+            <a href="https://medium.com/"
+              ><img src="icons/medium.svg" alt="medium icon"
+            /></a>
+          </li>
+          <li>
+            <a href="https://github.com/victormc13"
+              ><img src="icons/github.svg" alt="github icon"
+            /></a>
+          </li>
+          <li>
+            <a href="https://wellfound.com/"
+              ><img src="icons/angellist.svg" alt="angellist icon"
+            /></a>
+          </li>
+        </ul>
+      </footer>
```

Comment:
- [ ] According to the project requirements we must stick to the [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=1yfXx6yoiCqxC0tL-0) and make sure our design looks identical to figma. So please make sure your design also includes all the elements present in the figma design as your design is missing a line under the footer icons


Figma design:

<img width="231" alt="Screenshot 2023-03-29 at 8 14 04 AM" src="https://user-images.githubusercontent.com/44798044/228417732-c24824ee-e53c-4b7c-9f96-ac4b185dc75e.png">

Your design:

<img width="368" alt="Screenshot 2023-03-29 at 8 14 24 AM" src="https://user-images.githubusercontent.com/44798044/228417622-30962904-2eb9-4ad3-bd04-5c2963b612af.png">